src/renderer/store/mutations.js

```diff
@@ -44,6 +45,17 @@ export {
   startRequest,
 };
 
+const bigNumberToStringForStorage = (value) => {
+  if (_.has(value, 'c')) {
+    const number = new BigNumber(0);
+    number.c = value.c;
+    number.e = value.e;
+    number.s = value.s;
+    return number.toFixed(8);
```

Comment:
I guess the only problem here could be if some NEP5 tokens could support more than 8 digits before the decimal.